projects/packages/videopress/src/client/admin/components/admin-page/libraries.tsx

```diff
@@ -62,13 +62,19 @@ const VideoLibraryWrapper = ( {
 
 	const [ isFilterActive, setIsFilterActive ] = useState( false );
 
+	const singularTotalVideosLabel = __( 'Video', 'jetpack-videopress-pkg' );
+	const pluralTotalVideosLabel = __( 'Videos', 'jetpack-videopress-pkg' );
+	const totalVideosLabel = totalVideos === 1 ? singularTotalVideosLabel : pluralTotalVideosLabel;
```

Comment:
Since it relies on `totalVideos`, it could be `const singularTotalVideosLabel = __( '1 Video', 'jetpack-videopress-pkg' );` and
```js
const pluralTotalVideosLabel = sprintf(
	// translators: %$1s is the number of videos
	__( '%1$s Videos', 'jetpack-backup-pkg' ),
	totalVideos
);
```